### DISCUSSION TITLE
SendLater in -compose (command line)

### DISCUSSION THREAD

**Discussion by saidnai** [Extended-Thunder/send-later#407] [Ideas]
Hi,
Can I use SendLater in command ?
Like : 
`-compose from=mail@gmail.com,to=mail@outlook.com,subject=hi,body=test,sendlater=10mins`

Thanks for help ! :)

**Comment by jph-sendlater** [comment]
No, sorry.

**Reply by saidnai** [reply]
Thanks for your quick reply !
Maybe one day ? ^^

**Reply by jph-sendlater** [reply]
I don't think the way Send Later works is particularly conducive to this use case. Send Later requires that Thunderbird GUI be running in the background at the desired send time, in order to actually execute the send. If you're hoping to automate emails via scripts, you can just as well automate sending those emails directly via SMTP.

**Reply by saidnai** [reply]
> If you're hoping to automate emails via scripts, you can just as well automate sending those emails directly via SMTP.
This is what i'm doing ;)
You know how to do this ? ^^
